<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DemoLBPStaticImage</class>
 <widget class="QDialog" name="DemoLBPStaticImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>589</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>888</width>
    <height>589</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LBP-静态图像分析演示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="emotion_ai_main_window.qrc">
    <normaloff>:/EmotionAI/Resources/app1.png</normaloff>:/EmotionAI/Resources/app1.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>222</width>
     <height>260</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background:rgb(135, 167, 170);</string>
      </property>
      <property name="text">
       <string>原图</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="viewOriginalImageBtn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>view</string>
      </property>
      <property name="icon">
       <iconset resource="emotion_ai_main_window.qrc">
        <normaloff>:/cat/Resources/cat/catg.png</normaloff>:/cat/Resources/cat/catg.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_original">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(230, 230, 230);
font: 75 15pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>原图</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>222</width>
     <height>260</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_gray">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(230, 230, 230);
font: 75 15pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>灰度化</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="viewGrayImageBtn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>view</string>
      </property>
      <property name="icon">
       <iconset resource="emotion_ai_main_window.qrc">
        <normaloff>:/cat/Resources/cat/catg.png</normaloff>:/cat/Resources/cat/catg.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background:rgb(135, 167, 170);</string>
      </property>
      <property name="text">
       <string>灰度化</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>222</width>
     <height>260</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QPushButton" name="viewNormalImageBtn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>view</string>
      </property>
      <property name="icon">
       <iconset resource="emotion_ai_main_window.qrc">
        <normaloff>:/cat/Resources/cat/catg.png</normaloff>:/cat/Resources/cat/catg.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background:rgb(135, 167, 170);</string>
      </property>
      <property name="text">
       <string>尺寸归一化</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_normal">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(230, 230, 230);
font: 75 15pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>尺寸归一化</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>222</width>
     <height>260</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background:rgb(135, 167, 170);</string>
      </property>
      <property name="text">
       <string>检测人脸区域</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="viewFacedetectionImageBtn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>view</string>
      </property>
      <property name="icon">
       <iconset resource="emotion_ai_main_window.qrc">
        <normaloff>:/cat/Resources/cat/catg.png</normaloff>:/cat/Resources/cat/catg.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_facedetection">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(230, 230, 230);
font: 75 15pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>检测人脸</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="loadImageBtn">
      <property name="styleSheet">
       <string notr="true">background:rgb(174, 212, 91);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>加载图片</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>26</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="runImageBtn">
      <property name="styleSheet">
       <string notr="true">background:rgb(174, 212, 91);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>单面孔</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>26</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="runImageBtn_2">
      <property name="styleSheet">
       <string notr="true">background:rgb(174, 212, 91);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>多面孔</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>222</width>
     <height>260</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background:rgb(135, 167, 170);</string>
      </property>
      <property name="text">
       <string>人脸区域剪裁</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="viewFacecutImageBtn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>view</string>
      </property>
      <property name="icon">
       <iconset resource="emotion_ai_main_window.qrc">
        <normaloff>:/cat/Resources/cat/catg.png</normaloff>:/cat/Resources/cat/catg.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_facecut">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(230, 230, 230);
font: 75 15pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>人脸区域裁剪</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>222</width>
     <height>260</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;
background:rgb(135, 167, 170);</string>
      </property>
      <property name="text">
       <string>灰度直方图均衡化</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="viewGrayequalImageBtn">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>view</string>
      </property>
      <property name="icon">
       <iconset resource="emotion_ai_main_window.qrc">
        <normaloff>:/cat/Resources/cat/catg.png</normaloff>:/cat/Resources/cat/catg.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_gray_equal">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>220</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(230, 230, 230);
font: 75 15pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>灰度直方图均衡化</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>221</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="1">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">background:rgb(230, 230, 230);
font: 75 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>识别结果</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_result">
      <property name="styleSheet">
       <string notr="true">font: 75 18pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>what?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="emotion_ai_main_window.qrc">:/cat/Resources/cat/cata.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="emotion_ai_main_window.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>OnLoadImageBtn_Click()</slot>
  <slot>OnStartImageBtn_Click()</slot>
  <slot>ViewOriginalImage_Click()</slot>
  <slot>ViewGrayImage_Click()</slot>
  <slot>ViewGrayequalImage_Click()</slot>
  <slot>ViewFacedetectionImage_Click()</slot>
  <slot>ViewFacecutImage_Click()</slot>
  <slot>ViewNormalImage_Click()</slot>
  <slot>OnMultiFaceImageBtn_Click()</slot>
 </slots>
</ui>
